packages/strapi-plugin-content-manager/admin/src/components/DynamicZone/index.js

```diff
@@ -41,19 +42,27 @@ const DynamicZone = ({
   dynamicDisplayedComponents,
 }) => {
   const [isOpen, setIsOpen] = useState(false);
-
+  const [indexCategory, setIndexCategory] = useState(-1);
```

Comment:
I would like to change the logic a bit of the collapses. 

So here's my idea, we will use an object instead for the `dynamicComponentCategories`.

First let's change the useState hook:

```
const [categoryToOpen, setCategoryToOpen] = useState('');
```

Reply:
@soupette done in 15570e056d4c6c19e4615bd214dd63fbff7df881